### PR TITLE
Add PocketType

### PR DESCRIPTION
**Pull request by Maartenwut**
## Description

Adds the PocketType keyboard, a really small 12x4 ortholinear board

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/maartenwut/pockettype/pockettype.h [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#include "quantum.h"
+#define XXX KC_NO
```
Comment (zvecr) [inline keyboards/maartenwut/pockettype/pockettype.h:19]:
```suggestion
```

File: keyboards/maartenwut/pockettype/pockettype.c [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+	matrix_init_user();
+	led_init_ports();
+};
+
+void matrix_scan_kb(void) {
+	// put your looping keyboard code here
+	// runs every cycle (a lot)
+	matrix_scan_user();
+};
+
```
Comment (zvecr) [inline keyboards/maartenwut/pockettype/pockettype.c:30]:
```suggestion

```

File: keyboards/maartenwut/pockettype/pockettype.c [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "pockettype.h"
+
+void matrix_init_kb(void) {
+	// put your keyboard start-up code here
+	// runs once when the firmware starts up
+	matrix_init_user();
+	led_init_ports();
```
Comment (zvecr) [inline keyboards/maartenwut/pockettype/pockettype.c:22]:
```suggestion
	led_init_ports();

	matrix_init_user();
```

File: keyboards/maartenwut/pockettype/keymaps/default/readme.md [hunk 4]
```diff
@@ -0,0 +1 @@
+This is the default keymap for the Eon40.
```
Comment (zvecr) [inline keyboards/maartenwut/pockettype/keymaps/default/readme.md:1]:
```suggestion
# This is the default keymap for the pockettype.
```

File: keyboards/maartenwut/pockettype/rules.mk [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output on port C6
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
```
Comment (fauxpark) [inline keyboards/maartenwut/pockettype/rules.mk:32]:
```suggestion
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```

File: keyboards/maartenwut/pockettype/rules.mk [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+LAYOUTS = ortho_4x12
+LAYOUTS_HAS_RGB = no
```
Comment (fauxpark) [inline keyboards/maartenwut/pockettype/rules.mk:35]:
```suggestion
```

**Comment by fauxpark** [comment]
> Also needs a keyboard level readme

And an info.json.

**Comment by Maartenwut** [comment]
@fauxpark @zvecr 👍

File: keyboards/maartenwut/pockettype/rules.mk [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output on port C6
```
Comment (fauxpark) [inline keyboards/maartenwut/pockettype/rules.mk:29]:
```suggestion
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```

File: keyboards/maartenwut/pockettype/readme.md [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+
+## Support
+* Keyboard Maintainer: [Maarten Dekkers](https://github.com/maartenwut)
+* Hardware Supported: PocketType
+* Hardware Availability: [mechboards.co.uk](https://mechboards.co.uk/shop/kits/pockettype/)
+
+## Build
+Make example for this keyboard (after setting up your build environment):
+
+    make maartenwut/eon40:default
```
Comment (fauxpark) [inline keyboards/maartenwut/pockettype/readme.md:11]:
```suggestion
    make maartenwut/pockettype:default
```

**Comment by fauxpark** [comment]
Travis errors are unrelated.

**Comment by zvecr** [comment]
Thanks!

**Pull request merged**